### PR TITLE
chore: add issue template

### PR DESCRIPTION
**Pull request by jahnli**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: 🐞 Bug report
+description: Report an issue with Vue Languagse Tool
+labels: [untriaged]
```
Comment (so1ve) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: `pending triage`

Comment (so1ve) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]:
```suggestion
labels: ["pending triage"]
```

**Comment by so1ve** [comment]
Here is an existing PR: https://github.com/vuejs/language-tools/pull/4084

I suggest removing the `zh-CN` template since its functionality is similar to English's.

**Comment by jahnli** [comment]
`zh-CN` is only for the convenience of Chinese users to submit questions, and is not required

**Comment by jahnli** [comment]
Sorry, I don't see the existing PR, if you don't need to deliberately close it

**Comment by so1ve** [comment]
Nvm let's keep them open

**Comment by johnsoncodehk** [comment]
Thanks! I'll take a look at the form design later.

**Pull request closed without merging**